proto/rill/runtime/v1/queries.proto

```diff
@@ -408,6 +412,21 @@ message MetricsViewTimeSeriesResponse {
   repeated TimeSeriesValue data = 2;
 }
 
+message MetricsViewMeasureAlias {
+  enum MeasureType {
+    MEASURE_TYPE_UNSPECIFIED = 0;
+    MEASURE_TYPE_BASE_VALUE = 1;
+    MEASURE_TYPE_COMPARISON_VALUE = 2;
+    MEASURE_TYPE_ABS_DELTA = 3;
+    MEASURE_TYPE_REL_DELTA = 4;
+  }
+
+  string name = 1;
+  MeasureType type = 2;
+  repeated google.protobuf.Value args = 3;
+  string alias = 4;
+}
```

Comment:
1. Would incorporate `Comparison` in the name since this is specific to that use case (no other API supports base/comparison/delta values)
2. Should reuse (and probably rename) the `MetricsViewComparisonSortType` type so we don't have two enums doing the same